com.unity.mobile.notifications/Runtime/Android/AndroidNotificationCenter.cs

```diff
@@ -614,6 +614,7 @@ public class AndroidNotificationCenter
         /// The delegate type for the notification received callbacks.
         /// It is used in <see cref="AndroidNotificationCenter.OnNotificationReceived"/> event.
```

Comment:
Do we replace "see cref" here?

Reply:
Tooling doesn't complain about these. Strange. I'll leave them as is for now.